.github/workflows/test.yml

```diff
@@ -34,6 +34,12 @@ jobs:
             php-version: "7.2"
             prerequisites-script: install-bionic.sh
 
+          # Similar to Debian Bullseye
+          - runs-on: ubuntu-20.04
```

Comment:
Why add a 20.04 test and not a Debian 11 test? This will fail at the moment thanks to issues encountered in #1168

Reply:
Github Actions does not provide a Debian 11 platform to run on. So I tried to setup the most similar setup to what is bullseye.

I've been looking for a way to run all this test in docker images so we can pick ubuntu/debian/centos, but it requires some work, and it was part of the research in #1248.

Reply:
Ah I see, yeah that does make it tricky